frontend/src/components/worksheets/Worksheet/Worksheet.js

```diff
@@ -434,13 +440,35 @@ class Worksheet extends React.Component {
         }
     };
 
+    removeItemsFromSource = (itemIds) => {
+        let worksheetUUID = this.state.ws.uuid;
+        const url = `/rest/worksheets/${worksheetUUID}/add-items`;
+        $.ajax({
+            url,
+            data: JSON.stringify({ ids: itemIds }),
+            contentType: 'application/json',
+            type: 'POST',
+            success: () => {
+                const textDeleted = true;
+                const param = { textDeleted };
+                this.setState({ deleting: false });
+                this.reloadWorksheet(undefined, undefined, param);
+            },
+            error: (jqHXR, status, error) => {
+                this.setState({ deleting: false });
+                alert(createAlertText(this.url, jqHXR.responseText));
+            },
+        });
+    };
+
     confirmBundleRowAction = (code) => {
         if (
             !(
                 this.state.openDelete ||
                 this.state.openDetach ||
                 this.state.openKill ||
                 this.state.openCopy ||
+                this.state.openCut ||
```

Comment:
What about these? I don't think openCut is ever set?

Reply:
Ah good catch, forgot that we don't have dialogs anymore